chain/pool/txpool.go

```diff
@@ -587,9 +589,24 @@ func (tp *TxnPool) deleteTransactionFromMap(txn *transaction.Transaction) {
 }
 
 func (tp *TxnPool) CleanBlockValidationState(txns []*transaction.Transaction) error {
```

Comment:
I think this function wont return any error except nil, right?

Reply:
That's true, but in the future we might add error return to this function